modelarmor/quickstart/quickstart.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START modelarmor_quickstart]
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	modelarmor "cloud.google.com/go/modelarmor/apiv1"
+	modelarmorpb "cloud.google.com/go/modelarmor/apiv1/modelarmorpb"
+	"google.golang.org/api/option"
+)
+
+func main() {
+	// Google Project ID
+	projectID := "your-project-id"
+	// Google Cloud Location
+	locationID := "us-central1"
+	// ID For The Model Armor Template To Create
+	templateID := "go-template"
+
+	ctx := context.Background()
+	// Initialize Client
+	client, err := modelarmor.NewClient(ctx, option.WithEndpoint(fmt.Sprintf("modelarmor.%s.rep.googleapis.com:443", locationID)))
+	if err != nil {
+		fmt.Printf("Failed to create client: %v", err)
+	}
+	defer client.Close()
+	// Setup Template
+	parent := fmt.Sprintf("projects/%s/locations/%s", projectID, locationID)
+
+	template := &modelarmorpb.Template{
+		FilterConfig: &modelarmorpb.FilterConfig{
+			RaiSettings: &modelarmorpb.RaiFilterSettings{
+				RaiFilters: []*modelarmorpb.RaiFilterSettings_RaiFilter{
+					{
+						FilterType:      modelarmorpb.RaiFilterType_DANGEROUS,
+						ConfidenceLevel: modelarmorpb.DetectionConfidenceLevel_HIGH,
+					},
+					{
+						FilterType:      modelarmorpb.RaiFilterType_HARASSMENT,
+						ConfidenceLevel: modelarmorpb.DetectionConfidenceLevel_MEDIUM_AND_ABOVE,
+					},
+					{
+						FilterType:      modelarmorpb.RaiFilterType_HATE_SPEECH,
+						ConfidenceLevel: modelarmorpb.DetectionConfidenceLevel_HIGH,
+					},
+					{
+						FilterType:      modelarmorpb.RaiFilterType_SEXUALLY_EXPLICIT,
+						ConfidenceLevel: modelarmorpb.DetectionConfidenceLevel_HIGH,
+					},
+				},
+			},
+		},
+	}
+
+	req := &modelarmorpb.CreateTemplateRequest{
+		Parent:     parent,
+		TemplateId: templateID,
+		Template:   template,
+	}
+
+	createdTemplate, err := client.CreateTemplate(ctx, req)
+	if err != nil {
+		fmt.Printf("Failed to create template: %v", err)
```

Comment:
issue: use the %w formatting verb instead of %v when printing error objects.

Here and elsewhere.